tests/urdna2015/test060-in.nq

```diff
@@ -1,17 +1,38 @@
-<urn:ex:s> <urn:ex:000:empty> "" .
-<urn:ex:s> <urn:ex:001:simple> "simple" .
-<urn:ex:s> <urn:ex:002:quote> "\"" .
-<urn:ex:s> <urn:ex:003:backslash> "\\" .
-<urn:ex:s> <urn:ex:004:nl> "\n" .
-<urn:ex:s> <urn:ex:005:cr> "\r" .
-<urn:ex:s> <urn:ex:006:all> "\"\\\n\r" .
-<urn:ex:s> <urn:ex:007:uchar> "\u0022\u005c" .
-<urn:ex:s> <urn:ex:008:echar> "\t\b\n\r\f\"\'\\" .
-<urn:ex:s> <urn:ex:009> "\\u0039" .
-<urn:ex:s> <urn:ex:010> "\\n" .
-<urn:ex:s> <urn:ex:011> "\\\\" .
-<urn:ex:s> <urn:ex:012> "\"\"" .
-<urn:ex:s> <urn:ex:013> "\\\\\\" .
-<urn:ex:s> <urn:ex:014> "\"\"\"" .
-<urn:ex:s> <urn:ex:015> "\u221e" .
-<urn:ex:s> <urn:ex:016> "∞" .
+<_:s⁰> <_:p⁰> <_:o⁰> <_:g⁰> .
+<urn:ex:s:000:s\u20701> <urn:ex:000:p\u2070> <urn:ex:000:o\u2070> <urn:ex:000:g\u2070> .
+<urn:ex:s:000:s⁰2> <urn:ex:000:p⁰> <urn:ex:000:o⁰> <urn:ex:000:g⁰> .
+<urn:ex:s:001> <urn:ex:000:empty> "" .
+<urn:ex:s:001> <urn:ex:001:simple> "simple" .
+<urn:ex:s:001> <urn:ex:002:quote> "\"" .
+<urn:ex:s:001> <urn:ex:003:backslash> "\\" .
+<urn:ex:s:001> <urn:ex:004:nl> "\n" .
+<urn:ex:s:001> <urn:ex:005:cr> "\r" .
+<urn:ex:s:001> <urn:ex:006:all> "\"\\\n\r" .
+<urn:ex:s:001> <urn:ex:007:uchar> "\u0022\u005c" .
+<urn:ex:s:001> <urn:ex:008:echar> "\t\b\n\r\f\"\'\\" .
+<urn:ex:s:001> <urn:ex:009> "\\u0039" .
+<urn:ex:s:001> <urn:ex:010> "\\n" .
+<urn:ex:s:001> <urn:ex:011> "\\\\" .
+<urn:ex:s:001> <urn:ex:012> "\"\"" .
+<urn:ex:s:001> <urn:ex:013> "\\\\\\" .
+<urn:ex:s:001> <urn:ex:014> "\"\"\"" .
+<urn:ex:s:001> <urn:ex:015> "\u221e" .
+<urn:ex:s:001> <urn:ex:016> "∞" .
+<urn:ex:s:001> <urn:ex:017> <urn:ex:\u0065\u0078> .
+<urn:ex:s:001> <urn:ex:018> <urn:ex:\u221e> .
+<urn:ex:s:001> <urn:ex:019> <urn:ex:\u0020> .
```

Comment:
Although `<urn:ex:\u0020>` satisfies [IRIREF](https://w3c.github.io/rdf-n-quads/spec/#grammar-production-IRIREF), it does not result in a valid RFC 3987 IRI. When parsing, the UCHAR values are unescaped so this would include a space in an `isegment-nz-nc`.